src/components/StudyList/StudyList.js

```diff
@@ -53,6 +53,8 @@ const StudyList = ({studyList, loading, activeView = 'grid', user}) => {
       obj =>
         (obj.node.name &&
           obj.node.name.toLowerCase().includes(searchString.toLowerCase())) ||
+        (obj.node.kfId &&
+          obj.node.kfId.toLowerCase().includes(searchString.toLowerCase())) ||
```

Comment:
I think you could accomplish all this by concatenating all the fields, then seeing if it includes the string:
```
(obj.node.name + obj.node.shortName + obj.node.kfId).toLowerCase().includes(searchString.toLowerCase())
```